### PR TITLE
eth-json-rpc-infura@5.1.0

### PR DESCRIPTION
**Pull request by rekmarks**
This should be breaking for anyone that uses the Infura subprovider.

### CODE REVIEW

**Review by kumavis — APPROVED** [review]

**Review by kumavis — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "eth-block-tracker": "^4.4.2",
     "eth-json-rpc-errors": "^2.0.2",
     "eth-json-rpc-filters": "^4.2.1",
-    "eth-json-rpc-infura": "^4.0.1",
+    "eth-json-rpc-infura": "5.1.0",
```
Comment (Gudahtt) [inline package.json:29]: Was this meant to be pinned on `5.1.0` specifically? :thinking:

Comment (rekmarks) [reply package.json:29]: Boooo. No it was not. I'm trying to sneak in an unpinning here: https://github.com/MetaMask/web3-provider-engine/pull/357

**Comment by Ameerhamza12345** [comment]
hi

**Review by Ameerhamza12345 — APPROVED** [review]
hi

**Pull request merged**